<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,5">
    <property name="spacing">
     <number>40</number>
    </property>
    <property name="leftMargin">
     <number>40</number>
    </property>
    <property name="topMargin">
     <number>40</number>
    </property>
    <property name="rightMargin">
     <number>40</number>
    </property>
    <property name="bottomMargin">
     <number>40</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="left_plane" stretch="0,0,0">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="title">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="subtitle">
        <property name="text">
         <string>Presenter View</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="chart_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <layout class="QGridLayout" name="legend" columnstretch="0,0">
           <property name="leftMargin">
            <number>45</number>
           </property>
           <property name="topMargin">
            <number>45</number>
           </property>
           <property name="rightMargin">
            <number>45</number>
           </property>
           <property name="bottomMargin">
            <number>30</number>
           </property>
           <property name="horizontalSpacing">
            <number>15</number>
           </property>
           <property name="verticalSpacing">
            <number>20</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="legend_label_primary">
             <property name="text">
              <string>Text Label</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QFrame" name="legend_colour_primary">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>45</width>
               <height>45</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>45</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="legend_label_secondary">
             <property name="text">
              <string>Text Label</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QFrame" name="legend_colour_secondary">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>45</width>
               <height>45</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>45</width>
               <height>45</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="right_plane" stretch="0,1">
      <property name="spacing">
       <number>40</number>
      </property>
      <item>
       <widget class="QFrame" name="slide_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>35</number>
         </property>
         <property name="topMargin">
          <number>35</number>
         </property>
         <property name="rightMargin">
          <number>35</number>
         </property>
         <property name="bottomMargin">
          <number>35</number>
         </property>
         <item>
          <widget class="QLabel" name="slide">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>640</width>
             <height>360</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="questions_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>25</number>
         </property>
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>25</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>25</number>
         </property>
         <item>
          <widget class="QLabel" name="questions_title">
           <property name="text">
            <string>Participant Questions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="questions_box">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="questions_footer">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
